src/app/organizations/collection/collection.component.html

```diff
@@ -72,68 +66,93 @@ <h1>
+                  [routerLink]="[
+                    '/' + (entry.entryType === 'tool' ? 'container' : 'workflow') + 's/',
+                    entry.entryPath + (entry.versionName ? ':' + entry.versionName : '')
+                  ]"
+                >
+                  <div fxLayout="row" fxLayoutAlign="space-between">
+                    <div fxLayout="row" fxLayoutAlign="flex-start" fxLayoutGap="1rem">
+                      <img *ngIf="entry.entryType === 'workflow'" src="../../../assets/svg/workflow.svg" alt="workflow icon" />
+                      <img *ngIf="entry.entryType === 'tool'" src="../../../assets/svg/tool.svg" alt="tools icon" />
+                      <h5>
                         {{ entry.entryPath + (entry.versionName ? ':' + entry.versionName : '') }}
-                      </a>
-                    </h4>
-                  </mat-card-title>
-                  <mat-card-subtitle> Last updated {{ entry.dbUpdateDate | date }} </mat-card-subtitle>
-                </mat-card-header>
-                <mat-card-actions *ngIf="canEdit$ | async" fxLayout="row" fxLayoutAlign="end center">
-                  <span *ngIf="entry?.entryPath">
-                    <button
-                      mat-flat-button
-                      matTooltip="Remove entry from collection."
-                      id="removeEntryButton"
-                      (click)="
-                        openRemoveEntryDialog(organization.id, collection.id, entry.id, collection.name, entry.entryPath, entry.versionName)
-                      "
-                    >
-                      <mat-icon>clear</mat-icon> Remove
-                    </button>
-                  </span>
-                </mat-card-actions>
+                      </h5>
+                    </div>
+                    <div *ngIf="canEdit$ | async" fxLayout="row" fxLayoutAlign="end center">
+                      <span *ngIf="entry?.entryPath">
+                        <button
+                          mat-flat-button
+                          matTooltip="Remove entry from collection."
+                          id="removeEntryButton"
+                          (click)="
+                            openRemoveEntryDialog(
+                              organization.id,
+                              collection.id,
+                              entry.id,
+                              collection.name,
+                              entry.entryPath,
+                              entry.versionName
+                            )
+                          "
+                        >
+                          <mat-icon>clear</mat-icon> Remove
+                        </button>
+                      </span>
+                    </div>
+                  </div>
+                </a>
+                <hr />
+                <div fxLayout="row" fxLayoutAlign="space-between">
+                  <span class="update-time"> Last updated {{ entry.dbUpdateDate | date }} </span>
```

Comment:
The code on line 95 is checking if entry exists, but this code assumes it always does.

Reply:
@coverbeck I think I will just remove line 95, since no where else inside this file checks whether entry exists.  In addition to this, entry is an element inside the array, we could likely assume entry is not empty. 
Then take the anchor link somewhere else in this project for example, if entry does not exist then the redirection will fail. We could very well check if it exists and then determine whether anchor will be disabled accordingly, but like mentioned above, I don't think entry should be empty...